hack/multi-node/user-data.sample

```diff
@@ -25,30 +25,24 @@ coreos:
       content: |
         [Service]
         EnvironmentFile=/etc/environment
+        EnvironmentFile=/etc/kubernetes/kubelet.env
         Environment=KUBELET_ACI=quay.io/coreos/hyperkube
-        Environment=KUBELET_VERSION=v1.3.0_coreos.1
         ExecStartPre=/bin/mkdir -p /etc/kubernetes/manifests
         ExecStartPre=/bin/mkdir -p /srv/kubernetes/manifests
         ExecStartPre=/bin/mkdir -p /etc/kubernetes/checkpoint-secrets
-        ExecStart=/usr/lib/coreos/kubelet-wrapper \
-          --api-servers=https://172.17.4.101:443 \
-          --kubeconfig=/etc/kubernetes/kubeconfig \
-          --lock-file=/var/run/lock/kubelet.lock \
-          --exit-on-lock-contention \
-          --config=/etc/kubernetes/manifests \
-          --allow-privileged \
-          --hostname-override=${COREOS_PUBLIC_IPV4} \
-          --node-labels=master=true \
-          --minimum-container-ttl-duration=3m0s \
-          --cluster_dns=10.3.0.10 \
-          --cluster_domain=cluster.local
+        ExecStart=/usr/lib/coreos/kubelet-wrapper $PERSISTENT_KUBELET_OPTS $KUBELET_OPTS
         Restart=always
         RestartSec=5
 
         [Install]
         WantedBy=multi-user.target
 
 write_files:
+  - path: "/etc/kubernetes/kubelet.env"
+    content: |
+      KUBELET_OPTS='--api-servers=https://172.17.4.101:443 --kubeconfig=/etc/kubernetes/kubeconfig --lock-file=/var/run/lock/kubelet.lock --allow-privileged --config=/etc/kubernetes/manifests --minimum-container-ttl-duration=3m0s --cluster_dns=10.3.0.10 --cluster_domain=cluster.local'
+      PERSISTENT_KUBELET_OPTS='--exit-on-lock-contention=true --node-labels=master=true --hostname-override=$public_ipv4'
```

Comment:
Ah, I guess maybe you were suggesting we have a configMap the self-hosted uses and a configMap the on-host uses? That's a possibility too, but then there's 2 things to update instead of 1.